internal/scorer/algorithm/value_test.go

```diff
@@ -0,0 +1,169 @@
+// Copyright 2022 Criticality Score Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package algorithm
+
+import (
+	"testing"
+)
+
+func TestExistsCondition(t *testing.T) {
```

Comment:
I have combined `TestExistsCondition` and `TestNotCondition` into one test called `TestCondition`. But I don't understand what you mean by 
> "Rather than `"f"`, have `"condition"` and specify the condition in each `test` struct." 

Could you please clarify?